src/Akeneo/Category/front/src/feature/hooks/useEditAttributeTextAreaStatus.ts

```diff
@@ -0,0 +1,21 @@
+import {useRoute} from '@akeneo-pim-community/shared';
+import {useMutation} from 'react-query';
+import {apiFetch} from '../tools/apiFetch';
+
+export const useEditAttributeTextAreaStatus = (templateUuid: string, attributeUuid: string) => {
```

Comment:
```suggestion
export const useUpdateTemplateAttribute = (templateUuid: string, attributeUuid: string) => {
```
Suggesting name that match the endpoint and will accommodate for labels